### PR TITLE
Add unit tests for Browser component

### PR DESCRIPTION
**Pull request by tacigar**
**Before**

Before

**After**

After

### CODE REVIEW

File: zipkin-lens/src/components/Browser/BrowserHeader.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import BrowserHeader from './BrowserHeader';
+import { sortingMethods } from './sorting';
+
+describe('<BrowserHeader />', () => {
+  it('should render correct the number of results', () => {
```
Comment (codefromthecrypt) [inline zipkin-lens/src/components/Browser/BrowserHeader.test.js]: I am very happy you put in a "toehold test" like mentioned in @swankjesse's blog https://publicobject.com/2019/02/17/toehold-test/

this means later, we can test edge cases like zero results. Starting the first test class reduces burden of later change!

File: zipkin-lens/src/components/Browser/BrowserResults.test.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    width: 1,
+  };
+  const defaultProps = {
+    traceSummaries: [
+      { traceId: '1', duration: 5, ...traceSummaryCommonProps },
+      { traceId: '2', duration: 3, ...traceSummaryCommonProps },
+      { traceId: '3', duration: 7, ...traceSummaryCommonProps },
+    ],
+    sortingMethod: sortingMethods.LONGEST,
+    skewCorrectedTracesMap: {
```
Comment (codefromthecrypt) [inline zipkin-lens/src/components/Browser/BrowserResults.test.js]: this highlights to me if "skewCorrectedTracesMap" is a great name or not. clock skew is only one type of adjustment/correction. It might be helped to call this plainly tracesMap and differentiate something uncorrected as "raw traces"

File: zipkin-lens/src/components/Browser/sorting.test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { sortingMethods, sortTraceSummaries } from './sorting';
+
+const numberArrayToTraceSummaries = (numArray, key) => numArray.map(
+  n => ({ [key]: n }),
+);
+
+describe('sortTraceSummaries', () => {
+  it('LONGEST', () => {
```
Comment (codefromthecrypt) [inline zipkin-lens/src/components/Browser/sorting.test.js]: nit: for a test like this, sometimes I like to use tuples. for example (timestamp, duration). it is easier to see when there's a discrepancy between order when both values are literals also.

ex.

input = [
 (2, 1),
 (3, 2),
 (1, 3)
]
expect(sortedByTimestamp(input)).toEqual(
 (1, 3),
 (2, 1),
 (3, 2)
)

**Review by codefromthecrypt — APPROVED** [review]
good progress. I made some style comments which may not be relevant

File: zipkin-lens/src/components/Browser/BrowserResults.test.js (hunk 2 above)
Comment (tacigar) [reply zipkin-lens/src/components/Browser/BrowserResults.test.js]: I see. I'll rename them 👍

File: zipkin-lens/src/components/Browser/sorting.test.js (hunk 3 above)
Comment (tacigar) [reply zipkin-lens/src/components/Browser/sorting.test.js]: I see. Thank you for your advice.

**Comment by tacigar** [comment]
PTAL 🙇

**Review by codefromthecrypt — APPROVED** [review]
great
